src/web/src/components/TelescopeAvatar.tsx

```diff
@@ -20,23 +21,30 @@ const getInitials = (name: string) => {
   return initials;
 };
 
-const TelescopeAvatar = ({ name, img, size }: TelescopeAvatarProps) => {
+const TelescopeAvatar = ({ name, img, size, blog }: TelescopeAvatarProps) => {
   const initials = getInitials(name);
 
   return (
-    <Avatar
-      alt={name}
-      src={img}
+    <a
+      href={blog}
       style={{
-        width: size,
-        height: size,
-        fontSize: `calc(${size}px * 0.43)`,
-        backgroundColor: '#A0D1FB',
-        color: '#000',
+        textDecoration: 'none',
       }}
     >
-      {initials}
-    </Avatar>
+      <Avatar
+        alt={name}
+        src={img}
+        style={{
+          width: size,
+          height: size,
+          fontSize: `calc(${size} * 0.43)`,
```

Comment:
We might want to do this calculation on line 26 and store the result in const, then pass it down